### ISSUE TITLE
Change NSRL hashset look up module to use "known" flag as default

### ISSUE THREAD

**Issue by lfcnassif** [sepinf-inc/IPED#300]
Currently, files found in NSRL and not tagged in NSRL nor listed in KFFTaskConfig.txt as "alert" are tagged as "ignorable". That may not be true depending on use case. Furthermore, that makes updating NSRL harder, because the user should review the list of new softwares added to NSRL (often not tagged by them) to configure new "alert" programs in KFFTaskConfig.txt file, otherwise new files will be automatically tagged as "ignorable".

KFFTaskConfig.txt file will need to be refactored, to separate "alert" and "ignorable" programs and the new "known" default tag could be made configurable.

**Comment by tc-wleite** [comment]
I created a simple JSON file to configure mappings between NSRL product names and item status (hash:status).
There is an optional default status, for products not explicitly defined elsewhere.

I also created an option to "merge" or "skip", when the item status was already present (came from another hash set).
"skip" would do nothing, while "merge" will add the configured status for the product to the existing status (a multivalued field).

Multiple status definitions can be used, with any status value (not just alert, ignore or known), and different "whenPresent" settings. 

An example of the configuration file:
```json
{"defaultStatus": "known", 
 "defaultWhenPresent": "skip",
 "productStatus":
   [
    {"status": "alert", 
     "whenPresent": "merge",
     "productNames": 
        [
         "Hacker Tools",
	 "Hacks and Cracks"
	]
     },
    {"status": "ignore", 
     "whenPresent": "skip",
     "productNames": 
        [
         "Pinball Arcade"
	]
     }
   ]
}
```
